.openzeppelin/unknown-31337.json

```diff
@@ -0,0 +1,128 @@
+{
+  "manifestVersion": "3.2",
```

Comment:
add 31337 to gitignore

Reply:
done at https://github.com/rsksmart/swap-rbtc/pull/1/commits/5f89b46e915c40cef6db1b13ae1830cc66307eda